dm/ctl/master/update_validation.go

```diff
@@ -0,0 +1,87 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package master
+
+import (
+	"context"
+
+	"github.com/pingcap/tiflow/dm/ctl/common"
+	"github.com/pingcap/tiflow/dm/pb"
+	"github.com/spf13/cobra"
+)
+
+const (
+	UpdateValidationOp = "update"
+)
+
+type validationUpdateArgs struct {
+	sources  []string
+	taskName string
+
+	cutoverBinlogPos  string
+	cutoverBinlogGTID string
+}
+
+func NewUpdateValidationCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "update [task-name]",
+		Short: "update validation config of the completeness of the data",
+		RunE:  updateValidation,
+	}
+	cmd.Flags().String("cutover-binlog-pos", "", "specify the cutover binlog name for validation, should include binlog name and pos in brackets, e.g. '(mysql-bin.000001, 5989)'")
+	cmd.Flags().String("cutover-binlog-gtid", "", "specify the cutover binlog gtid for validation, only valid when source config's gtid is enabled, e.g. '1642618e-cf65-11ec-9e3d-0242ac110002:1-30'")
+	return cmd
+}
+
+func updateValidation(cmd *cobra.Command, _ []string) error {
+	args, msg, ok := parseValidationUpdateArgs(cmd)
+	if !ok {
+		return printUsageAndFailWithMessage(cmd, msg)
+	}
+	// we use taskName="" to represent we do operation on all task, so args.allTask don't need to pass
```

Comment:
we have a err `"`task-name` should be set"`, what this comment mean, pass task-name as `""`?

Reply:
the argguments are just copied from other functions. already removed